### ISSUE TITLE
WOF crashes on boot / GNR no flashers & others same...

### ISSUE THREAD

**Issue by mazjohn** [vpinball/pinmame#247]
VpinMame build 880 with latest VPX x64 build 1923, Flashers not working on GNR, Family Guy, WPT, & others - table crashes on load with WOF.. Roms up to date- Just wondering- will backward compatibility be priority or will creators need to update table code for impacted tables? thank you..

**Comment by francisdb** [comment]
Regarding WOF, lamp arrays in the script need to be extended to 610 items
see #196

**Comment by mazjohn** [comment]
Thank you. I don't normally post here-  If it helps at all to know, - Switching back to earlier build 720 of VpinMame resolves all issues:  no flasher issues with all tables and the Flippers no longer intermittently lock up, which was also occurring on latest VpinMame build 888.  All testing done with latest VPX X64 build 1923.  Kind regards.

**Comment by toxieainc** [comment]
> Just wondering- will backward compatibility be priority or will creators need to update table code for impacted tables? thank you..

In general, we try to stay as backwards compatible as feasible with PinMAME. But for some few things we decided to 'break' things, for example if it improves emulation, e.g. like its the case with WOF.
The rest you describe sounds like a bug to me, but @vbousquet is the expert there.

**Comment by vbousquet** [comment]
As far as I can tell, all the other bugs have been solved (GNR is pending an update, it's a known table bug. Medieval Madness also needs an update which is not plan which is annoying but as far as I can tell this is also a table bug and the fix is available).
To be honest, I would say, I'm fairly surprised that in the end, with all the deep changes done, nearly everything is backward compatible.

**Comment by mazjohn** [comment]
> @mazjohn WOF,GNR,MM need the table tweaks as mentioned, the flashers should be back to normal with latest commit, please verify.

@toxieainc   Feedback after 1 hour of light testing: Updated to latest VpinMame commit build 896 running with latest VPX X64 build 923.  not noticing any issues other then the aforementioned needed table tweaks- The Only exception being WOF, being the flippers are still getting stuck intermittently in the up position. Not sure if that's still a separate legitimate issue or not from the flashers, but worth mentioning- I willl test more variety of tables the next few days, and report back if anything else noticed. thank you all for the amazing work done here on this project over the years - it truly is god level stuff you all do for us...

**Comment by mazjohn** [comment]
@toxieainc  :Some meant to be hopefully taken as constructive feedback-  As a follow-up to my above response from yesterday ;  After more extensive testing of VpinMame build 896 combined with VPX X64 build 923, I have concluded that there are many tables experiencing crashes during load or after starting a game with ball 1.   An example of this is with VPW's ACDC Luci table, where it throws  Runtime error
------------- Line: 4202 Overflow: '[number: 34680]'   after shooting ball 1- --  I've reverted back to VpinMame build 720 with the same latest VPX X64 build 923, and the issues go away.  It seems in all of my testing for backward compatibility, I conclude that if backward compatibility was my primary concern, that the last usable beta build of VpinMame for me was **build 720** dated Dec31,2023. That build has no flasher issues, table crashes, or overblown lighting...  and where I am holding at for time being.
